fmts/ruby.go

```diff
@@ -27,7 +27,9 @@ func init() {
 	register(&Fmt{
 		Name: "reek",
 		Errorformat: []string{
-			`%*\s%f:%l: %m`,
+			`%-P%f -- %t warnings:`,
+			`%*\s[%l, %c]:%m`,
+			`%*\s[%l]:%m`,
 			`%-G%.%#`,
 		},
 		Description: "(reek --single-line) Code smell detector for Ruby",
```

Comment:
I'm guessing the difference is because of `--single-line` flag?
I'm not familiar with reek, so can you confirm? @alirezabashyri @mgrachev

Reply:
Yes my bad.